scripts/fix_encumbrances/fix_encumbrances_quesnelia.py

```diff
@@ -0,0 +1,965 @@
+#!/usr/bin/env python
```

Comment:
for reviewers: the easiest way to make review - checkout this branch in Idea and use Ctrl+D to comare 2 files, previous script and current script